packages/expo-tracking-transparency/src/TrackingTransparency.ts

```diff
@@ -111,4 +111,13 @@ export function isAvailable(): boolean {
   );
 }
 
+/**
+ * Returns the [AAID](https://developer.android.com/training/articles/ad-id) on Android and [IDFA](https://developer.apple.com/documentation/adsupport/asidentifiermanager/advertisingidentifier) on iOS.
+ *
+ * @example `"e9228286-4c4e-4789-9d95-15827dcb291b"`
+ */
+export async function getAdvertisingIdAsync(): Promise<string | undefined> {
```

Comment:
If the native function has a return type, it will never return or resolve to `undefined` - it would be `null` instead.

```suggestion
export async function getAdvertisingIdAsync(): Promise<string | null> {
```